src/stories/interfaces.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+export interface ButtonProps {
+  onClick: () => void;
+  children: React.ReactNode;
+  classes?: string;
+  icon?: JSX.Element;
+}
+
+export interface IconProps {
+  iconLink: string;
+}
```

Comment:
I prefer placing types for props alongside the component (either in the same file or in the component folder). Having a single file for all of your types may quickly make it messy and difficult to navigate

Reply:
So should I split 
`scr/stories/interfaces.tsx`
to the 
`scr/stories/Button/interfaces.tsx`,
`scr/stories/Icon/interfaces.tsx`
?